tests/unit/note-route.test.js

```diff
@@ -30,6 +29,7 @@ const note = Object.assign(new Note(), {
     type: 'Note',
     content: 'Sample Note content',
     'oa:hasSelector': {},
+    pubcontext: 'http://localhost:8080/publication-abc123',
```

Comment:
Guessing that this is a typo or copy-paste flub since this is the same url as 'context'?

Other than that, looks good.

Reply:
that was a temporary fix to the problem of 'context' disappearing from the Note object. I did find a better fix, so that should have been removed.

Reply:
Ah, that makes sense. 👍